integration-tests/smoke/automation_test.go

```diff
@@ -763,26 +855,43 @@ func TestAutomationKeeperNodesDown(t *testing.T) {
 			t.Parallel()
 			l := logging.GetTestLogger(t)
 			config, err := tc.GetConfig("Smoke", tc.Automation)
-			if err != nil {
-				t.Fatal(err)
-			}
+			require.NoError(t, err, "Failed to get config")
+
 			a := setupAutomationTestDocker(
 				t, registryVersion, automationDefaultRegistryConfig(config), false, false, &config,
 			)
 
-			consumers, upkeepIDs := actions.DeployConsumers(
+			sb, err := a.ChainClient.Client.BlockNumber(context.Background())
+			require.NoError(t, err, "Failed to get start block")
+
+			consumers, upkeepIDs := actions_seth.DeployConsumers(
 				t,
+				a.ChainClient,
 				a.Registry,
 				a.Registrar,
 				a.LinkToken,
-				a.Deployer,
-				a.ChainClient,
 				defaultAmountOfUpkeeps,
 				big.NewInt(automationDefaultLinkFunds),
 				automationDefaultUpkeepGasLimit,
 				false,
 				false,
 			)
+
+			t.Cleanup(func() {
```

Comment:
can we abstract clean up logic into a function?

Reply:
+1